### PR TITLE
CI for openSUSE Tumbleweed

### PR DESCRIPTION
**Pull request by Werkov**
<del>Don't merge, only draft pull request so far.</del>

The series is ready for review now. The openSUSE CI still requires changes to mkosi posted in systemd/mkosi#758, so I kept the external-todo commit in this branch to have a working example (the final merge should not include these but this may be blocked on new mkosi version release). Further details are explained in respective commits.

### CODE REVIEW

**Comment by Werkov** [comment]
The 'mkosi / ci (debian)' failure (`Timed out waiting for device /dev/gpt-auto-root.`) should be unrelated to my changes (it's presumably systemd/mkosi#725).

File: .github/workflows/mkosi.yml [hunk 1]
```diff
@@ -21,10 +21,11 @@ jobs:
           - debian
           - ubuntu
           - fedora
+          - opensuse
 
     steps:
     - uses: actions/checkout@v2
-    - uses: systemd/mkosi@v9
+    - uses: Werkov/mkosi@opensuse-github
```
Comment (mrc0mmand) [inline .github/workflows/mkosi.yml]: Is still change still relevant, since https://github.com/systemd/mkosi/pull/758 was merged?

Comment (Werkov) [reply .github/workflows/mkosi.yml]: A change of this step is still relevant. The reference to v9 wouldn't work whereas pointing to the `main` branch would IMO be unstable (not reproducible test) and affecting multiple systemd PRs. The clearer solution to me seems to point the upcoming mkosi version tag.

 @DaanDeMeyer what are plans for tagging a new version of mkosi (for the reasons above)? (BTW thanks for merging systemd/mkosi#758.)

Comment (bluca) [reply .github/workflows/mkosi.yml]: A new release has been done, can you update this now?

**Review by bluca — APPROVED** [review]
LGTM

File: .github/workflows/mkosi.yml (hunk 1 above)
Comment (Werkov) [reply .github/workflows/mkosi.yml]: Done and thanks!

**Pull request merged**